project/README.md

```diff
@@ -45,13 +45,13 @@ Cookieplone will ask a lot of questions.
 You can use some of its options to avoid repeatedly entering the same values.
 
 
-#### `--no-input`
+#### `--no_input`
 
-You can use the [`--no-input`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-no-input) option to make the cookiecutter not prompt for parameters and only use `cookiecutter.json` file content.
+You can use the [`--no_input`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-no-input) option to make the cookieplone not prompt for parameters and only use `cookiecutter.json` file content.
 
 
 #### `--replay` and `--replay-file`
-You can use the options [`--replay`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-replay) and [`--replay-file`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-replay-file) to not prompt for parameters and only use information entered previously or from a specified file.
+You can use the option [`--replay-file`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-replay-file) to not prompt for parameters and only use information entered previously or from a specified file.
```

Comment:
Why was the `--replay` option removed?

If it should be removed, then you should also remove " entered previously or".

Reply:
I plan to fix that on cookieplone

Reply:
Sorry, I'm not clear. What is the fix? Restore `--replay` or remove its associated text?

Reply:
Restore --replay in cookieplone